CONTRIBUTORS

```diff
@@ -316,7 +311,7 @@ Shruti Satish <shrutisatish00@gmail.com>
 Shubha Gupta <shubha1593@gmail.com>
 Shubha Rajan <shubharajan94@gmail.com>
 Shubham Bansal <shubhambansal17@hotmail.com>
-Shuta Suzuki <shutas@umich.edu>
+Shuta Suzuki <shutas@umich.edu>Aadya Mishra <aadyamishra24@gmail.com>
```

Comment:
I'm not sure how this happened. `Aadya Mishra <aadyamishra24@gmail.com>` is not  present in release_summary.md. Should I add it to the right place or remove the name?

Reply:
@ankita240796, Do you have any thoughts on how this happened? Is there any issue with the release script? (Just wanted to make sure that our release script is working fine!) 

Note: I'm considering these changes were made automatically by the release script. @aks681 Can you please confirm?

Reply:
Yup, this was automatic. I've now removed this name here since they are already present in all the files.

Reply:
Can you run the script `update_changelog_and_credits` by commenting out branch push, to confirm what is happening? I am not able to reproduce this behaviour.